src/popup.html

```diff
@@ -7,8 +7,8 @@
   <link rel="shortcut icon" href="./icons/favicon.ico" type="image/x-icon">
```

Comment:
This path needs to be changed to something like ./assets/icons/favicon.ico because the image is missing
And the same on line 7, 16